editor/import/resource_importer_scene.cpp

```diff
@@ -924,6 +945,8 @@ Node *ResourceImporterScene::_pre_fix_node(Node *p_node, Node *p_root, HashMap<R
 			print_verbose(vformat("Fix: Renamed %s to %s", original_path, new_path));
 			r_node_renames.push_back({ original_path, p_node });
 		}
+		//If we created new node instead, merge meta values from the original node
```

Comment:
```suggestion
		// If we created new node instead, merge meta values from the original node.
```

Reply:
Applied, but note the file predominantly uses this style of comments though.